posts/Yoann/2017-06-12-hibernate-search-5-8-0-Beta3.adoc

```diff
@@ -0,0 +1,238 @@
+but if you are an integrator, or simply if you feel like it,
+you can have a look at our integration tests:
+
+ * Wildfly:
+   https://github.com/hibernate/hibernate-search/tree/bf4342c8991e0384243ff22317c87102738daf59/integrationtest/wildfly/src/test/java/org/hibernate/search/test/integration/wildfly/cdi/integration[the integration per se],
+   and https://github.com/hibernate/hibernate-search/blob/bf4342c8991e0384243ff22317c87102738daf59/integrationtest/wildfly/src/test/resources/cdi/META-INF/services/org.hibernate.service.spi.ServiceContributor[a Java service to plug the integration into Hibernate Search].
+ * Spring Boot:
+   https://github.com/hibernate/hibernate-search/tree/bf4342c8991e0384243ff22317c87102738daf59/integrationtest/spring/src/test/java/org/hibernate/search/test/integration/spring/injection/integration[the integration per se],
+   https://github.com/hibernate/hibernate-search/blob/bf4342c8991e0384243ff22317c87102738daf59/integrationtest/spring/src/test/resources/META-INF/services/org.hibernate.service.spi.ServiceContributor[a Java service to plug the integration into Hibernate Search],
+   and https://github.com/hibernate/hibernate-search/blob/bf4342c8991e0384243ff22317c87102738daf59/integrationtest/spring/src/test/java/org/hibernate/search/test/integration/spring/injection/SpringInjectionITApplicationConfiguration.java[the configuration of the bean factory].
+
+=== And more!
+
+For a full list of changes since 5.8.0.Beta2, please see https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10061&version=28701[the release notes].
+
+Below is a summary of other notable changes:
+
+ * https://hibernate.atlassian.net/browse/HSEARCH-2736[HSEARCH-2606]: the `discovery_scheme` configuration
+   property is now correctly taken into account.
+   Thanks to Matthieu Vincent for reporting and fixing this issue.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2477[HSEARCH-2477]: shard filtering now works on Elasticsearch.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2603[HSEARCH-2603]: we now use the Painless scripting language
+   when doing spatial searches on Elasticsearch 5+.
+   Incidentally, this means server-side configuration is not required anymore to perform spatial search on Elasticsearch 5+.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2734[HSEARCH-2734]: due to a lot of incorrect (harmful) use,
+   we have deprecated the "ram" directory provider.
+   If you need it, please **ensure you are not using it a production environment**,
+   read about https://docs.jboss.org/hibernate/search/5.8/reference/en-US/html_single/#directory-provider-table[its limitations in the reference documentation],
+   and use its new name: "local-heap".
+ * https://hibernate.atlassian.net/browse/HSEARCH-2735[HSEARCH-2735]: index-time boosting features (`@Boost`, `@DynamicBoost`)
+   have been deprecated with no replacement,
+   and will be removed when Hibernate Search will switch to Lucene 7.
```

Comment:
Technically it hasn't (yet) because Lucene 7 hasn't been released. I'll rephrase with something like "will need to be removed in a future version because Lucene 7 won't allow index-time boosting anymore".